config.toml

```diff
@@ -0,0 +1,84 @@
+# Configuring Nameshed
+# ====================
+
+# The configuration file version.
+# - 'v1': This format.
+version = "v1"
+
+# Configuring the Nameshed daemon.
+#
+# This controls Nameshed's behaviour as a daemon, i.e. how it interacts with the
+# external system.
+[daemon]
+#
+# The minimum severity of the messages logged by the daemon.
+#
+# Messages at or above the specified severity level will be logged.  The
+# following levels are defined:
+# - 'trace': A function or variable was interacted with, for debugging.
+# - 'debug': Something occurred that may be relevant to debugging.
+# - 'info': Things are proceeding as expected.
+# - 'warning': Something does not appear to be correct.
+# - 'error': Something went wrong (but Nameshed can recover).
+# - 'critical': Something went wrong and Nameshed can't function at all.
+log-level = "info"
+#
+# The location the daemon writes logs to.
+#
+# Logs are appended line-by-line to the specified file path.  It can be set to
+# '/dev/stdout' or '/dev/stderr' for standard output and error, respectively.
+# If it is a terminal, ANSI escape codes may be used to style the output.
+log-file = "/dev/stdout"
+
+# Configuring how zones are loaded.
+[loader]
+# Where to listen for zone update notifications.
+#
+# A DNS server will be bound to these addresses.  If a DNS NOTIFY message for a
+# known zone is received there, the zone will be reloaded appropriately.
+#
+# Unless explicitly specified (e.g. 'udp://localhost:0000'), each address will
+# be served over UDP and TCP.  An empty array will disable listening entirely.
+notif-listeners = ["127.0.0.1:8050"] # TODO: Pick a good default port
```

Comment:
If you're going to use 5 out of the 6 letters of notify I would use the entire word.